samples/Directory.Packages.props

```diff
@@ -1,54 +1,54 @@
 <Project ToolsVersion="15.0">
-	<PropertyGroup>
-		<ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-	</PropertyGroup>
-	<ItemGroup>
-		<PackageVersion Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-		<PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
-		<PackageVersion Include="Microsoft.Extensions.Logging.Debug" Version="5.0.0" />
-		<PackageVersion Include="Microsoft.Net.Compilers.Toolset" Version="4.7.0-2.final" />
-		<!--
+  <PropertyGroup>
```

Comment:
Please keep tabs instead of space please

Reply:
fixed